integration-tests/src/test/java/com/datastax/oss/driver/core/metadata/NodeStateIT.java

```diff
@@ -148,13 +148,13 @@ public void setup() {
             driverContext.getLoadBalancingPolicy(DriverExecutionProfile.DEFAULT_NAME);
 
     // Sanity check: the driver should have connected to simulacron
-    ConditionChecker.checkThat(
+    await()
+        .alias("Connections established")
```

Comment:
do you know why some `await()` have an alias, and others don't?

Reply:
The alias is just an informational message to make it easier to understand the error. We've not been using it consistently in our tests.